br/pitr-known-issues.md

```diff
@@ -58,3 +64,27 @@ Issue: [#13126](https://github.com/tikv/tikv/issues/13126)
 After a network partition failure in the cluster, the backup task cannot continue backing up logs. After a certain retry time, the task will be set to `ERROR` state. At this point, the backup task has stopped.
 
 To resolve this issue, you need to manually execute the `br log resume` command to resume the log backup task.
+
+## The actual storage space used by log backup is 2~3 times that of the incremental data displayed in the cluster monitoring metrics
+
+Issue: [#13306](https://github.com/tikv/tikv/issues/13306)
+
+This issue occurs because the monitoring metrics show the size of data after RocksDB compaction, while log backup stores KV data in a customized encoding format. The different storage methods lead to different data compression ratios, the difference of which is 2~3 times.
```

Comment:
```suggestion
This issue log backup data use a customized encoding format. The different format lead to different data compression ratios, the difference of which is 2~3 times.
```

Reply:
```suggestion
This issue occurs because log backup data use a customized encoding format. The different format leads to different data compression ratios, the difference of which is 2~3 times.
```